### PR TITLE
[WIP] chore(demo-integrations): `Editor` stabilize cypress-tests

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [X] Build or CI related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by nsbarsukov** [comment]
1 successful workflow run ✅

File: projects/demo-integrations/cypress/support/commands.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ declare global {
 
 Cypress.Commands.add('getByAutomationId', id => cy.get(`[automation-id=${id}]`));
 Cypress.Commands.add('findByAutomationId', {prevSubject: true}, (subject: any, id) =>
-    subject.find(`[automation-id=${id}]`),
+    cy.wrap(subject).find(`[automation-id=${id}]`),
```
Comment (nsbarsukov) [inline projects/demo-integrations/cypress/support/commands.ts:27]: `subject` is just `jQuery`-element which has method `.find()`. 
**This method doesn't have [retry-ability](https://docs.cypress.io/guides/core-concepts/retry-ability#Commands-vs-assertions)!**
___
`cy.wrap(subject)` is a `$Chainer`-element (cypress built-in implementation) which **also** has method `.find()`.
**This method has [retry-ability](https://docs.cypress.io/guides/core-concepts/retry-ability#Commands-vs-assertions)!**

**Review by splincode — APPROVED** [review]

File: projects/demo-integrations/cypress/support/commands.ts (hunk 1 above)
Comment (splincode) [reply projects/demo-integrations/cypress/support/commands.ts:27]: add a comment to the code

**Comment by nsbarsukov** [comment]
successful run ✅

**Comment by splincode** [comment]
it turns out this test always falls, I wonder what's wrong with it

image

**Comment by nsbarsukov** [comment]
3 successful runs in a row ✅

**Comment by nsbarsukov** [comment]
4 successful runs in a row ✅
5st time was fail because of the icon
icon-fail

**Pull request closed without merging**